index.src.html

```diff
@@ -1411,6 +1426,10 @@
 : <dfn event for=VideoEncoder>dequeue</dfn>
 :: Fired at the {{VideoEncoder}} when the {{VideoEncoder/encodeQueueSize}} has
     decreased.
+    
+: <dfn event for=VideoEncoder>onEncoderChange</dfn>
+:: Fired at the {{VideoEncoder}} when the {{VideoEncoder/powerEfficient}} or
+    {{VideoEncoder/decoderId}} value changes.
```

Comment:
Are these valid references? I don't find an attribute for powerEfficeint on the VideoEncoder or AudioEncoder. The only powerEfficient reference I find is to MediaCapabilities, but that is the capability of a query, not what is used by an encoder atm.